exercises/proverb/src/main/java/Proverb.java

```diff
@@ -0,0 +1,12 @@
+class Proverb {
+    private String[] words;
```

Comment:
You don't need this in the starter implementation. You only need enough so that the tests will compile :)